openlibrary/templates/account/books.html

```diff
@@ -29,6 +29,9 @@
 $elif key == 'already-read':
   $ og_title = _("Books %(username)s has read", username=userDisplayName)
   $ og_description = _("%(username)s has read %(total)d books. Join %(username)s on OpenLibrary.org and tell the world about the books that you care about.", username=userDisplayName, total=total_items)
+$elif key == 'stopped-reading':
+  $ og_title = _("Books %(username)s have stopped reading", username=userDisplayName)
+  $ og_description = _("%(username)s have stopped reading %(total)d books.", username=userDisplayName, total=total_items)
```

Comment:
```suggestion
  $ og_title = _("Books %(username)s has stopped reading", username=userDisplayName)
  $ og_description = _("%(username)s has stopped reading %(total)d books.", username=userDisplayName, total=total_items)
```
Small grammar correction.